### ISSUE TITLE
Undefined is not an object (evaluating 'rows[j][k].toString')

### ISSUE THREAD

**Issue by natdm** [facebook/react-native#5050]
When I'm debugging in chrome and I click on a menu item that I've made, it runs the function like I coded. When I'm NOT debugging, I get 'Undefined is not an object (evaluating 'rows[j][k].toString')'.

https://youtu.be/6nsPr-2T6QE

https://github.com/natdm/PanResponderApp/blob/master/FullSwipe.ios.js#L172-L204

**Comment by brentvatne** [comment]
Hi there! This looks like a bug in the `console.table` polyfill - https://github.com/facebook/react-native/blob/4cb04315e7de835b52cbdc2ae3072ddaadb9283b/packager/react-packager/src/Resolver/polyfills/console.js#L399-L460

PR welcome :)

**Comment by mihirsoni** [comment]
@brentvatne  Hi, 

I have created PR for the same , but I found one more issue with it. Not sure whether it is required to take care.

We fetch the columns only for the [first row of an array] (https://github.com/facebook/react-native/blob/4cb04315e7de835b52cbdc2ae3072ddaadb9283b/packager/react-packager/src/Resolver/polyfills/console.js#L417) , what if row 1 , 2 has additional columns, do we care for that or not ?
```javascript
var rows = [{
            color: 'red'
        }, {
            color: 'green',
            height: '10'
        ];
```
In above example , the console table will only display color and not both.
